modules/nf-core/quartonotebook/main.nf

```diff
@@ -69,17 +58,28 @@ process QUARTONOTEBOOK {
         params_cmd = dumpParamsYaml(nb_params)
         render_args = "--execute-params params.yml"
     }
+    ( params_cmd ?
```

Comment:
I meant for that format to replace the whole functions, but I can make PR for that.